classloader/src/main/java/io/smallrye/common/classloader/resource/InMemoryResource.java

```diff
@@ -0,0 +1,64 @@
+package io.smallrye.common.classloader.resource;
```

Comment:
Should `classloader` really be part of the package?

Reply:
Agree. I even doubt this should be in the `classloader` module, there should be an extra module for this IMHO.

Reply:
Well, I tried to get away with it :)